tools/parse_baselines/providers/types.py

```diff
@@ -101,10 +101,12 @@ def parse(self) -> dict:
 
```

Comment:
aren't there any other changes needed for the baseline parsers? I'm thinking of statistic names, which changed greatly in this PR (e.g. from `value` to `result` and many other were renamed to `Avg`)